packages/components/src/select-control/README.md

```diff
@@ -18,17 +18,17 @@ SelectControl allow users to select from a single-option menu. It functions as a
 
 Use a select control when:
 
-- You want users to select a single option from a list.
+- You want users to select one or more options from a list.
 - There is a strong default option.
 - There is little available space.
 - The contents of the hidden part of the menu are obvious from its label and the one selected item. For example, if you have an option menu labelled “Month:” with the item “January” selected, the user might reasonably infer that the menu contains the 12 months of the year without having to look.
 
-If you have a shorter list of options, or need all of the options visible to the user, consider using RadioControl instead.
```

Comment:
I removed this because the way the multiselects are designed by browsers, all of the options are visible to the user, or at least will have a vertical scroll so as to scroll through the options. With the `multiple` attribute, there is no dropdown as in the single selection.